### ISSUE TITLE
Book Creator Sample app giving bug

### ISSUE THREAD

**Issue by lakshya-mishra** [microsoft/semantic-kernel#1542]
While creating the “book Creator app”locally, I’m getting the same errors as shown in the attachments, continuously.

**Additional context**
There are almost 3 repeated errors that I am getting and those are as follows:

1) SyntaxError: expected property name or
‘}’ in JSON at position 2

2) SyntaxError: Unexpected token…….

3) Error: internal server error =>

**NOTE**
The first two are very common in the last one comes in between them and rarely I get the desired output..

PS :- I am trying to run in the latest code.

![IMG_7023](https://github.com/microsoft/semantic-kernel/assets/74200472/9c941cc2-c160-411b-baee-48ebf3ed9b11)

![IMG_7024](https://github.com/microsoft/semantic-kernel/assets/74200472/b0e1a7a7-21f3-45cb-bfca-c630e6acb9ad)

![IMG_7026](https://github.com/microsoft/semantic-kernel/assets/74200472/ec3b8918-2a1f-4c88-8a11-0f119a044fc2)

![IMG_7029](https://github.com/microsoft/semantic-kernel/assets/74200472/76a3c703-e905-409a-85ed-c900fda038f1)

**Comment by lakshya-mishra** [comment]
@teresaqhoang But I'm using chat completion model only to run this app as instructed in the Readme file,  i.e., "gpt-3.5-turbo."

And how to fix the errors I'm getting while parsing the JSON? Those SyntaxErrors.

**Comment by teresaqhoang** [comment]
Hey @lakshya-mishra, we're testing the sample locally and running into unexpected behavior with SequentialPlanner. 

In the interim while we stabilize this sample, I have some changes that updates the planner to fix the Syntax errors as well as use ActionPlanner in this PR: https://github.com/microsoft/semantic-kernel/pull/1613. Feel free to pull the changes in this PR locally using `gh pr checkout 1613` to unblock yourself while we get a solution published. 

If you have access to gpt-4, we recommend using that as it serves a more stable Planner experience. 

And we are looking into another issue related to Planner XML syntax errors. https://github.com/microsoft/semantic-kernel/issues/1483

I'll update this thread with any progressions!

**Comment by lakshya-mishra** [comment]
Thankyou @teresaqhoang for the clarifications and acknowledging the issue, till the issues are fixed, will continue as suggested…

**Comment by teresaqhoang** [comment]
Hey @lakshya-mishra, the PR has been merged, so I'm closing out this issue. 

I've created a separate issue to track SequentialPlanner support in our samples, so you can track that development there: https://github.com/orgs/microsoft/projects/852/views/6?filterQuery=Sequential&pane=issue&itemId=31417114
